docs/games/0002-StardewValley.md

```diff
@@ -138,6 +142,59 @@ Regardless of platform, similarly to Bethesda Script Extenders, SMAPI comes with
 - Windows (Steam and GOG): `StardewModdingAPI.exe` can be launched directly. Alternatively, [Steam](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Windows#Steam) and [GOG Galaxy](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Windows#GOG_Galaxy) can be configured to launch `StardewModdingAPI.exe` instead of the original game executable `Stardew Valley.exe`. This is only needed if you want achievements and playtime tracking to work.
 - Windows ([Xbox Game Pass](https://stardewvalleywiki.com/Modding:Installing_SMAPI_on_Windows#Xbox_app)): the original game executable `Stardew Valley.exe` has to be replaced with `StardewModdingAPI.exe` for mods to work.
 
+## Mod Conflicts
+
+### File Conflicts
+
+Before SMAPI Content Packs and the Content Patcher were a thing, assets had to be changed by replacing the raw XNB files in the `Content` directory. Most XNB mods have been [migrated](https://forums.stardewvalley.net/threads/migrating-xnb-mods-to-content-patcher-packs.564/) to the Content Patcher format, but some older mods didn't receive an "official" update. The wiki contains a [list of unofficial updates](https://stardewvalleywiki.com/Modding:Using_XNB_mods#Alternatives_using_Content_Patcher).
+
+Two mods that replace the same XNB file are in conflict with each other and the order in which they are deployed determines the winner.
+
+### C# Mods
+
+C# Mods can target an incompatible version of SMAPI or of the game itself. Conflicts between SMAPI mods are logic/functionality based.
```

Comment:
For design it might be good to have a list of possible incompatibilities:
- Can a mod require a newer version of SMAPI?
- Can a mod require an older version of SMAPI (or is forward compatibility guaranteed?)
- Same for Game versions
- Same for Mod dependencies versions
- Can mods define other mods they are incompatible with in the manifest?

Reply:
Mods can use the `manifest.json` to define a [minimum SMAPI version](https://stardewvalleywiki.com/Modding:Modder_Guide/APIs/Manifest#Minimum_SMAPI_version). Since SMAPI versions are tied to game versions, you indirectly specify what game versions the mod supports.

Reply:
SMAPI also offers a mod compatibility list: https://smapi.io/mods

Reply:
Ok, so it assumes that a mod working with version 1.0 of SMAPI will work on 1.1, and same for mod dependencies. This is good info to know. Could you add it to the doc? Same for the Game being indirect dependency.

SMAPI ver-> Game ver on the other hand is exact? So new game version is not compatible usually?